server/requirements.txt

```diff
@@ -19,4 +19,5 @@ redis==3.5.3
 opencensus
 opencensus-ext-stackdriver
 opencensus-ext-flask
-typing_extensions
\ No newline at end of file
+typing_extensions
+Werkzeug==1.0.1
```

Comment:
@beets @chejennifer this is the cause.. https://pypi.org/project/Werkzeug/#history

Reply:
nice find!